DEVELOPMENT.md

```diff
@@ -118,6 +118,22 @@ kubectl create clusterrolebinding cluster-admin-binding \
   --user="${USER}"
 ```
 
+### Install in custom namespace
+1. To install into a different namespace you will need to modify resources in the `./config`
+    - remove all `namespace: tekton` resources
```

Comment:
nit: Can you update this to  "remove all `namespace: tekton` references"

Reply:
done